### PR TITLE
proxy/httpproxy: fix v2 proxy log header

### PR DESCRIPTION
**Pull request by gyuho**
Replace all with capnslog

### CODE REVIEW

File: proxy/httpproxy/reverse.go [hunk 1]
```diff
@@ -95,11 +94,11 @@ func (p *reverseProxy) ServeHTTP(rw http.ResponseWriter, clientreq *http.Request
 
 	endpoints := p.director.endpoints()
 	if len(endpoints) == 0 {
-		msg := "proxy: zero endpoints currently available"
+		msg := "httpproxy: zero endpoints currently available"
```
Comment (heyitsanthony) [inline proxy/httpproxy/reverse.go]: s/httpproxy://?

File: proxy/httpproxy/reverse.go [hunk 2]
```diff
@@ -152,9 +151,9 @@ func (p *reverseProxy) ServeHTTP(rw http.ResponseWriter, clientreq *http.Request
 
 	if res == nil {
 		// TODO: limit the rate of the error logging.
-		msg := fmt.Sprintf("proxy: unable to get response from %d endpoint(s)", len(endpoints))
+		msg := fmt.Sprintf("httpproxy: unable to get response from %d endpoint(s)", len(endpoints))
```
Comment (heyitsanthony) [inline proxy/httpproxy/reverse.go]: s/httpproxy://?

**Comment by gyuho** [comment]
@heyitsanthony Fixed. PTAL. Thanks!

File: proxy/httpproxy/reverse.go [hunk 3]
```diff
@@ -95,11 +95,11 @@ func (p *reverseProxy) ServeHTTP(rw http.ResponseWriter, clientreq *http.Request
...
 	endpoints := p.director.endpoints()
 	if len(endpoints) == 0 {
-		msg := "proxy: zero endpoints currently available"
+		msg := "zero endpoints currently available"
 		reportRequestDropped(clientreq, zeroEndpoints)
 
 		// TODO: limit the rate of the error logging.
-		log.Printf(msg)
+		plog.Println(msg)
 		e := httptypes.NewHTTPError(http.StatusServiceUnavailable, msg)
```
Comment (heyitsanthony) [inline proxy/httpproxy/reverse.go]: `"httpproxy: " + msg`?

File: proxy/httpproxy/reverse.go [hunk 4]
```diff
@@ -152,9 +152,9 @@ func (p *reverseProxy) ServeHTTP(rw http.ResponseWriter, clientreq *http.Request
 
 	if res == nil {
 		// TODO: limit the rate of the error logging.
-		msg := fmt.Sprintf("proxy: unable to get response from %d endpoint(s)", len(endpoints))
+		msg := fmt.Sprintf("unable to get response from %d endpoint(s)", len(endpoints))
 		reportRequestDropped(clientreq, failedGettingResponse)
-		log.Printf(msg)
+		plog.Println(msg)
 		e := httptypes.NewHTTPError(http.StatusBadGateway, msg)
```
Comment (heyitsanthony) [inline proxy/httpproxy/reverse.go]: `"httpproxy: " + msg`?

**Comment by gyuho** [comment]
All addressed. PTAL.

Thanks!

**Comment by heyitsanthony** [comment]
lgtm

**Comment by gyuho** [comment]
thanks. merging in greens.

**Pull request merged**